guess_the_number.js

```diff
@@ -0,0 +1,41 @@
+import rl from 'readline-sync'
+
+const theTruth = Math.floor( Math.random() * 100 ) + 1
+
+const isValid = input => {
+  return  ( parseInt( input ) === true ) ||
+          ( parseInt(input) < 101 && parseInt(input) > 0 )
+}
+
+const getInput = () => {
+  let guess
+  do {
+      guess = rl.question( 'A number between 1 and 100 please: ' )
+      if ( guess === 'exit' )
+        return guess
+  } while( !isValid( guess ) )
+
+  return parseInt(guess)
```

Comment:
Consistent formatting should be used - if you use spaces in parens, they should be used consistently.  Consider the use of a linting tool to help reinforce your team's code style.